wasm/gas.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
How are we generating this file, exactly?

Reply:
At the moment manually. We first run the benchmarks, then we go through the html report produced and compute the gas cost based on the average execution time. The conversion should be something like 1ms~10 gas units. I didn't feel like automating this because we only need to run the benchmarks from time to time